README.md

```diff
@@ -122,6 +122,12 @@ You can optionally create an `.eslintignore` file to ignore file paths. The `.es
 **/coverage/*
 ```
 
+If you want to ignore the same files and folders contained in the `.gitignore` file, you can do so by omitting the creation of the `.eslintignore` file and using the following command for linting instead:
```

Comment:
The new ignore text, doesn't seem to scan well.

Is there a way to make this, more succinct and improve its clarity, without the need for some many 'the's?

Reply:
Sorry @Alicole -- Ben approved the PR before I saw your comment. What alternative would you suggest?